src/iblt.h

```diff
@@ -29,6 +29,8 @@ SOFTWARE.
 #include <set>
 #include <vector>
 
+static const size_t VALS_32 = 4294967295;
+
```

Comment:
I think it would be better to use:

static const size_t VALS_32 = std::numeric_limits<unsigned long>::max()

Reply:
Sorry @ptschip, this is cruft too. Removing...

Reply:
I think these extraneous variables are popping up because I rebased so many times.